people/ondrej-bojar.md

```diff
@@ -5,8 +5,8 @@ title: Ondřej Bojar
 description: Machine translation researcher and associate professor
 ---
 
-**Ondřej Bojar** is a machine translation researcher and since 2010 an associate professor at the [Institute of Formal and Applied Linguistics](https://ufal.mff.cuni.cz/) at Charles University in Prague.
-His work on [statistical](/statistical-machine-translation) and later [neural machine translation](/neural-machine-translation) is widely cited.
+**Ondřej Bojar** is a machine translation researcher at the [Institute of Formal and Applied Linguistics](https://ufal.mff.cuni.cz/) at Charles University in Prague.
+He became an associate professor in 2019 and his work on [statistical](/statistical-machine-translation) and later [neural machine translation](/neural-machine-translation) is widely cited.
```

Comment:
I think the link should be as follows:

`and his work on [statistical](/../approaches/statistical-machine-translation.md) and later [neural machine translation](/../approaches/neural-machine-translation.md)...`

We have created an issue #238 to fix links, but for the time being, this is the way they are working.